apps/storybook-react/tailwind.css

```diff
@@ -85,6 +85,6 @@
 @font-face {
   font-family: 'MMPoly';
   font-style: normal;
-  font-weight: 700;
+  font-weight: 600;
```

Comment:
Because we use only one font weight we have to also update MMPoly and MMSans. This doesn't effect the weight because we are still pointing to the same font files. See Before/After in description